src/server/src/main/docker/Dockerfile

```diff
@@ -97,26 +97,26 @@ ADD ${SHADED_JAR} /usr/local/lib/cassandra-reaper.jar
 ADD spreaper /usr/local/bin/spreaper
 
 
-# get around `/usr/local/bin/configure-persistence.sh: line 65: can't create /etc/cassandra-reaper.yml: Interrupted system call` unknown error
-RUN touch /etc/cassandra-reaper.yml
-# get around `/usr/local/bin/configure-webui-authentication.sh: line 44: can't `create` /etc/shiro.ini: Interrupted system call` unknown error
-RUN touch /etc/shiro.ini
+# get around `/usr/local/bin/configure-persistence.sh: line 65: can't create /etc/reaper/cassandra-reaper.yml: Interrupted system call` unknown error
+RUN touch /etc/reaper/cassandra-reaper.yml
+# get around `/usr/local/bin/configure-webui-authentication.sh: line 44: can't `create` /etc/reaper/shiro.ini: Interrupted system call` unknown error
+RUN touch /etc/reaper/shiro.ini
```

Comment:
This looks inconsistent with line 108 and 122 where the `shiro` directory is created under `/etc/cassandra-reaper` 

Reply:
Good find.  There were 3 locations related to shiro here that I'll update.